beginner_setup_guide.md

```diff
@@ -0,0 +1,164 @@
+
+It is recommended to download the additional data for the GB-Printer-Web interface. To automatically download the additional data do:
+
+#### Windows
+Go to your Start Menu, type ```powershell``` and navigate to the directory you just extracted (e.g. cd ```YourUsername\Downloads\websocket-gbp-emulator\```)
+Run the download script with ```.\update_w.ps1```
+
+#### MacOS / Linux
+Open your Terminal and navigate to the files you just extracted (e.g. ```cd ~/Downloads/websocket-gbp-emulator/```)
+Type ```chmod +x update_w.sh```
+Type ```./update_w.sh```
+
+### 1.5 Upload the code and data
+
+Connect your ESP8266 Board via USB to your PC. Navigate to ```File -> Open ...``` navigate to the folder you unzipped in the previous step. Open the ```websocket-gbp-emulator.ino``` 
+
+To confirm that the code was downloaded correctly navigate to ```Sketch -> Verify/Compile```. If everything works go to ```Sketch -> Upload``` and wait until the upload is finished.  
+
+Now nagivate to ```Tools -> ESP8266 Sketch Data Upload``` and wait until its finished.
+
+### 1.6  Test the board
+
+Unplug your ESP8266 Board from your PC and replug it. Open your WiFi Minitor and wait for it to finish scanning for available networks. You should see a wifi network named ```gameboyprinter``` with the password ```gameboyprinter```. 
+
+After you connected to the WiFi open your browser and type in the URL ```http://gameboyprinter.local```. The Gameboy Printer Web Emulator should load. If it doesn't please try to connect to ```https://192.168.4.1``` instead. 
+
+## Step 2: Preparing the Hardware
+
+Now that you've got your ESP8266 running with the websocket-gbp-emulator it's time to connect your Gameboy to it to actually copy your images. There are non-destructive ways ([Example](https://github.com/Palmr/gb-link-cable)) but for ease of access we will have to cut up a link cable. You can either obtain those cheap from ebay or find reproductions on websites like AliExpress. 
+
+### 2.1 Link Cable connection without a multimeter
+
+#### 2.1.1 Preperation
+Place your prepared ESP8266 Microcontroller on your breakout board.
+
+Take your Link Cable and cut it in half. You will find six (6) thin wires inside. Put one half aside and remove the housing around the connector carfully with a knife on the other side until you can see the connected wires. 
+
+![Link Cable Connection Wires](/docs/link_cable_connection_wires.jpg)
+
+**IMPORTANT: The wire colors shown here can be different from your cable.**
+
+Set this cable aside for now and take the other half. Carefully remove the cable shielding for 2cm / 1 inch close to where you cut the cable to expose the six wires.
+
```

Comment:
I think the cables are "cross-link" so looking at the other cable might actually cause confusion?